app/questionnaire/questionnaire_store_updater.py

```diff
@@ -152,7 +152,7 @@ def remove_primary_person(self, list_name: str) -> None:
             self.remove_list_item_and_answers(list_name, list_item_id)
 
     def remove_list_item_and_answers(self, list_name: str, list_item_id: str) -> None:
-        """Remove answers from the answer store and update the list store to remove it.
+        """Remove answers from the answer store, remove list item progress form the progress store and update the list store to remove it.
```

Comment:
I don't see any changes that removes the progress from this method?

Reply:
There isn't a code change, I updated the comment to reflect what the function was already doing -  on line 169 it removes all progresses associated with the given list item (repeating sections and list item progresses)

Reply:
Ah right. While we are here, update the method name to reflect that?.

Reply:
Updated 👍